### ISSUE TITLE
Have time-varying input for sex ratio at birth

### ISSUE THREAD

**Issue by shajiang93** [IvanWilli/DemoKin#34]
Now the SRB is fixed across countries and years.

**Comment by IvanWilli** [comment]
Yes. We can think in accepting a vector, equal length than years.  In case some difference, complete constant in the right.
I think this is safer than add a data.frame indicating years

**Comment by IvanWilli** [comment]
Hi @shajiang93,
What do you mean with countries? the ratio is fixed but you can set it in the function argument

**Comment by IvanWilli** [comment]
now can be a time vector
